app/controllers/enrollments/organisations_controller.rb

```diff
@@ -20,7 +20,8 @@ def update
 
     def load_defaults
       @organisation = Organisation.find(params[:id])
-      @enrollment = FloodRiskEngine::Enrollment.find_by(token: params[:enrollment_id])
+      @enrollment = FloodRiskEngine::Enrollment.find_by(token: params[:enrollment_id]) ||
+                    FloodRiskEngine::Enrollment.find(params[:enrollment_id])
```

Comment:
Why is it that the enrollment_id in params can be both an id and a token?

Reply:
Good spot @jjromeo - enrollment.id was hardcoded into a presenter path. Removed now so it always looks up by token.